proxy/envoy/resources.go

```diff
@@ -429,6 +436,15 @@ type Cluster struct {
 	outbound bool
 }
 
+// SslCntext definition
+// This is used for both Listener::ssl_context and Cluster::ssl_context.
```

Comment:
Not exactly, verifySAN is used only for cluster::ssl_context. May need to adjust the comment.